app/views/NewMessageView/UserItem.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { memo } from 'react';
+
+import { themes } from '../../lib/constants';
+import { ISearch, TSubscriptionModel } from '../../definitions';
+import UserItemComponent from '../../containers/UserItem';
+import { useTheme } from '../../theme';
+import { TGoRoomItem } from '../../lib/methods/helpers/goRoom';
+import sharedStyles from '../Styles';
+
+type TItem = ISearch | TSubscriptionModel;
+
+export interface IUserItem {
+	item: TItem;
+	index: number;
+	search: TItem[];
+	chats: TSubscriptionModel[];
+	useRealName: boolean;
+	goRoom: (item: TGoRoomItem) => void;
+}
+
+const UserItem = memo(
+	({ item, index, chats, search, useRealName, goRoom }: IUserItem) => {
```

Comment:
chats and search are wrong here.
You're adding complexity to the component and coupling it to the parent view when you could just send the data you actually want as props.
It could be just a single `listLength` prop.

Theory aside, I couldn't find why this is still needed.
Couldn't it just be a `ListFooterComponent`?
When we're doing refactors like this, we need to actually evaluate the code and understand what needs to be improved.

Reply:
As we talked about yesterday, I removed this component and implemented it directly in the render item.